homework-submission/src/myList.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class  myList extends Component {
```

Comment:
Use PascalCase (`MyList`) for both class name and file name. A better name would be `TodoList`.